backend/src/test/java/com/cruru/evaluation/domain/repository/EvaluationRepositoryTest.java

```diff
@@ -42,8 +43,8 @@ void sameIdUpdate() {
     @Test
     void saveNoId() {
         //given
-        Evaluation evaluation1 = new Evaluation(2, "서류가 인상 깊었습니다.", null, null);
-        Evaluation evaluation2 = new Evaluation(5, "포트폴리오가 인상 깊었습니다.", null, null);
+        Evaluation evaluation1 = createEvaluation();
+        Evaluation evaluation2 = createEvaluation();
```

Comment:
```suggestion
        Evaluation evaluation2 = createEvaluation();
```
```suggestion
        Evaluation evaluation = createEvaluationExcellentPorfolio();
```

Evaluation도 여러 Fixture 나눠보는 것은 어떠신가요?

Reply:
반영하였습니다.